src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
```

Comment:
No need to set container height - it's defined by content.

Reply:
Noted for future compliance with thanks.